bindings/wasm/examples-account/src/node.ts

```diff
@@ -32,6 +34,8 @@ async function main() {
             return await unchecked();
         case "multiple_identities":
             return await multipleIdentities();
+        case "storage_test_suite":
+            return await storageTestSuite(async () => new MemStore())
```

Comment:
No, you're right. I had it like that initially, but changed it so the example could be called from `stronghold-nodejs`. But since that ended up needing its own custom test calls (to exclude the `didPurge` test), combining it with `memory_storage` is better.